fastcrypto/src/tests/hmac_tests.rs

```diff
@@ -81,26 +81,25 @@ fn test_regression_of_salt_padding() {
 
 #[test]
 fn test_regression_of_hmac() {
-    // Using test vectors from https://datatracker.ietf.org/doc/html/rfc4231#page-3
```

Comment:
are you still using any test cases from standard? or these are just generated?

Reply:
no, unfortunately there aren't ones in the rfc